docs/manual.tex

```diff
@@ -213,21 +222,26 @@ \subsection{Division}
 whose variant is the product of the inputs' variants, whose dimension is the quotient of the inputs' dimensions, and whose numerical representation
 type is that of the inputs.
 
-Specialized to operate on quantities or on units, the type of division is again less intimidating:
+Specialized to operate on quantities, the type of division is again less intimidating:
 
 \begin{lstlisting}
 (/) :: (Fractional a) => Quantity d1 a -> Quantity d2 a -> Quantity (d1 / d2) a
+\end{lstlisting}
+
+Ditto for units:
```

Comment:
Woah, re-reading that type just uncovered a serious bug in my fixed point support!